src/helm/benchmark/window_services/http_model_window_service.py

```diff
@@ -0,0 +1,27 @@
+from .local_window_service import LocalWindowService
+from .tokenizer_service import TokenizerService
+
+
+class HTTPModelWindowServce(LocalWindowService):
```

Comment:
Add a todo to remove this soon... this isn't generally useful because `tokenizer_name` is hardcoded to `neurips/local` (which we are also removing soon).